test/E2ETest/TestHelpers.cs

```diff
@@ -16,6 +16,7 @@
 using Xunit;
 using Xunit.Abstractions;
 using Microsoft.Tye;
+using YamlDotNet.RepresentationModel;
```

Comment:
pref: make a new class and call it `YamlAssert`. That follows the pattern of xUnit.

This is why I always hate on "Helpers" - they accumulate a bunch of clutter 😆